ppr-api/src/ppr_api/models/search_utils.py

```diff
@@ -147,10 +150,10 @@
        r.registration_number AS base_registration_num,
        CASE WHEN p.last_name = :query_last AND
                  p.first_name = :query_first AND
-                 p.middle_initial = :query_middle THEN 'EXACT' ELSE 'SIMILAR' END match_type,
+                 p.middle_initial = 'E' THEN 'EXACT' ELSE 'SIMILAR' END match_type,
```

Comment:
I think this might be a typo? Probably want this to match what they queried right?

Reply:
Thanks Kial, yes I do. Updated.